RecoLocalMuon/GEMRecHit/test/runGEMRecHitProducer_cfg.py

```diff
@@ -42,6 +42,14 @@
 process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
```

Comment:
Hi Piet,

If there are good reasons to have the other (non-RPC) files updated,
I don't see a need to split the PR.
I just wanted to check that those files were forward ported consistently


On 4/13/15 10:00 AM, Piet Verwilligen wrote:
> In RecoLocalMuon/GEMRecHit/test/runGEMRecHitProducer_cfg.py
> <https://github.com/cms-sw/cmssw/pull/8524#discussion_r28245606>:
> 
>> @@ -42,6 +42,14 @@
>>  process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
> 
> Thanks for having a look.
> The goal was to have all the digitization and local reco working for all
> muon detectors for the upgrade (CSC+DT+GEM+RPC). That is why i was using
> (and modifying) this file. I will split the pull request in two:
> 1)one part dedicated to fixing the RPCDitizer;
> 2) one to part containing all other changes to have the digi and local
> reco working for all muon detectors for the upgrade
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/8524/files#r28245606>.
> 

-- 
***********************************************************************
 Vyacheslav (Slava) Krutelyov
 TAMU: Physics Dept Texas A&M MS4242, College Station, TX 77843-4242
 CERN: 42-R-027
 AIM/Skype: siava16              googleTalk: slava77@gmail.com
 (630) 291-5128  Cell (US)       +41 76 275 7116 Cell (CERN)

-----------------------------------------------------------------------